umbraco-deploy/deploy-settings.md

```diff
@@ -260,3 +263,27 @@ If you would prefer to use SQL Server LocalDb when it's available on your local
         }
     }
 ```
+
+## MediaFileChecksumCalculationMethod
+
+Deploy will do comparisons between the entities in different environments to determine if they match and decide whether to include them in the operation. By default, for media files, a check is made on the a portion of the intial bytes of the file.
+
+This corresponds to the default setting of `PartialFileContents`.
+
+If a lot of files need to checked, this can be slow, and a faster option is available that uses the file metadata. The only downside of changing this option is a marginally increased the chance of Deploy considering a media file hasn't changed when it has.  This would omit it from the deployment.
```

Comment:
```suggestion
If a lot of files need to be checked, this can be slow, and a faster option is available that uses the file metadata. The only downside of changing this option is a marginally increased chance of Deploy considering a media file hasn't changed when it has.  This would omit it from the deployment.
```